src/renderer/pages/Settings/Networks/model/networks-overview-model.ts

```diff
@@ -39,6 +49,57 @@ sample({
   target: [activeNetworksModel.events.networksChanged, inactiveNetworksModel.events.networksChanged],
 });
 
+sample({
+  clock: networksFilterModel.$filteredNetworks,
+  target: [activeNetworksModel.events.networksChanged, inactiveNetworksModel.events.networksChanged],
+});
+
+sample({
```

Comment:
I would say you don't need comments if the code explicitly describes what it does or you're get used to read samples
1. read clock - `addCustomRpcModel.output.flowFinished`
2. read target - `networkSelectorModel.events.rpcNodeSelected`
Then if you need more details look into `filter / fn`
If code is messy or case is actually unique and requires human explanation - than yes, I up to leave a comment

Or even with such scheme of reading samples you'd better have comments?
It might end up having comments near every sample, which leads to questions: "is it a bad and unreadable code?" or "reading effector code is just a pure pain 😢" or ... "it's a matter of habit"

Reply:
As someone is new to a code base, it's always easier to read plain text comments, and while I'm just thinking about any newcomer or outsider. Lowering the barrier to read code is often better. Whlie I agree that it's not absolutely needed for every sample, if you felt like it'd help me, or anyone reviewing this PR by adding this comment on Github, then I thought why not extend this help to any later reader.

Reply:
@sokolova-an suggests to create a document for contributors where we should describe vision of the project (not business value), how to interact with code in terms of FSD, how to read effector declarative operators an so on.
I'm strongly influenced by Robert Martin and his vision of a "clean code", so I'd better write a single document with guidelines instead of leaving comments here and there. And for sure continue code decoupling with the help of FSD.
My goal is to make code nice and tidy, comments are a good signal that code is too big to occupy just enough space in your brain.